src/parser/transforms.js

```diff
@@ -268,7 +268,7 @@ module.exports.preprocessGlimmerTemplates = function preprocessGlimmerTemplates(
         let codeSlice = code.slice(...n.range);
         for (const part of n.tag.split('.')) {
           let pattern = `\\b${part}\\b`;
-          if (part.startsWith(':')) {
+          if (part.startsWith(':') || part.startsWith('@')) {
```

Comment:
I should probably extend this to any special character?
A tag could also be `_Foo`

Reply:
Or just use indexOf, that should work...

Reply:
Difference is _Foo could be defined locally